eng/scripts/docs/type2docfx.version.txt

```diff
@@ -0,0 +1 @@
+2.0.8
```

Comment:
Does this need to be an independent file? Or should we just hardcode it in the ps1 script or snippet? Not a hung deal either way but if it is only used by those then perhaps, we don't need a separate file.

Reply:
If we hardcode it in a ps1 file then automation has to update the value in the file without breaking the rest of the script. If the hypothetical script file changes, the update logic should also not break it. I think it's worth using the separate file to prevent extra complexity in the update logic.